build.zsh.in

```diff
@@ -473,27 +477,15 @@ function main {
 
 		typeset -a distfiles
 		distfiles=($(get_distfiles))
-		local i
-
-		for i in {1..${#distfiles[@]}}; do
-			if duplicated "${distfiles[${i}]}" "${distfiles[@]}"; then
-				distfiles[${i}]=
-			fi
-		done
+		local i flag
+        distfiles=( ${(u)distfiles})
+        write DISTFILES= ${(j: \\\n\t:):-'${PACKAGE}-${VERSION}/'$^distfiles} "\n"
```

Comment:
Indentation.

Also, Makefile style: please add a space between the variable and the `=` sign.